pkg/apis/maistra/conversion/security_test.go

```diff
@@ -1034,3 +1034,79 @@ func TestSecurityConversionFromV2ToV1(t *testing.T) {
 		})
 	}
 }
+
+type multiClusterTestCase struct {
+	name               string
+	spec               *v2.ControlPlaneSpec
+	expectedHelmValues v1.HelmValues
+}
+
+var multiClusterTestCases = []multiClusterTestCase{
+	{
+		name: "MultiCluster_v2_4",
+		spec: &v2.ControlPlaneSpec{
+			Version: versions.V2_4.String(),
+			Security: &v2.SecurityConfig{
+				CertificateAuthority: &v2.CertificateAuthorityConfig{
+					Type: v2.CertificateAuthorityTypeCertManager,
+					CertManager: &v2.CertManagerCertificateAuthorityConfig{
+						Address: "",
+					},
+				},
+			},
+		},
+		expectedHelmValues: buildHelmValues(`
+global:
+  caAddress: 
+meshExpansion:
+  enabled: false
+useILB: false
+multiCluster: 
+  enabled: false
+multiClusterOverrides:
+  expansionEnabled:
+multiClusterEnabled:
+meshExpansion:
+  enabled: false
+multiCluster:
+  enabled: false
+multiClusterEnabled:
+multiClusterOverrides:
+  expansionEnabled:
+spec:
+  cluster:
+    multiCluster:
+      enabled: true
+  gateways:
+    enabled: false
+    useILB: false
+`),
```

Comment:
These values are obviously incorrect, so this test fails as expected.

Reply:
When I run the test locally it is asking me to add a bunch of stuff, should I keep it to what you had put into the jira?

Reply:
> When I run the test locally it is asking me to add a bunch of stuff

You should never adjust expected test data to the actual result just to make the test pass. Your test and data should drive the implementation, so first fix the expected helm values and the input (SMCP spec) and then work on the implementation until it passes for the correct input and output.

> should I keep it to what you had put into the jira?

In JIRA I added only SMCP spec. You can use that YAML to properly configure input SMCP in your test.